toolchain/sem_ir/formatter.cpp

```diff
@@ -665,7 +665,17 @@ class Formatter {
   template <typename NodeT>
   auto FormatInstructionRHS(NodeT node) -> void {
     // By default, an instruction has a comma-separated argument list.
-    std::apply([&](auto... args) { FormatArgs(args...); }, node.args_tuple());
+    using Info = TypedNodeArgsInfo<NodeT>;
+    if constexpr (Info::NumArgs == 2) {
+      auto tuple = StructReflection::AsTuple(node);
+      FormatArgs(std::get<Info::Arg0Field>(tuple),
+                 std::get<Info::Arg0Field + 1>(tuple));
```

Comment:
Would it be worth defining `Info::Arg1Field`?

Reply:
Sure, done.